src/components/Flex/Flex.stories.tsx

```diff
@@ -0,0 +1,100 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import React from 'react';
+import { Meta, StoryFn } from '@storybook/react';
+import { Flex, FlexProps } from './Flex';
+import { Box } from '@components/Box';
+import { BOX_BG_ALIAS_TOKENS_CLASSES, BOX_TEXT_ALIAS_TOKENS_CLASSES } from '@utilities/tokens';
+import { SPACING_VALUES } from '@utilities/dimensions';
+
+const DirectionTypes = ['row', 'col', 'row-reverse', 'col-reverse'];
```

Comment:
You must use the same values as defined on the types in the component file because with these values the property control is not working. `['row', 'column', 'row-reverse', 'column-reverse'];`